eos_data_distribution/ndn/http.py

```diff
@@ -57,14 +57,13 @@ def read_bytes_async():
     read_bytes_async()
 
 
-def fetch_http_headers(session, url):
+def fetch_http_headers(session, url, callback):
```

Comment:
although it's a hack because it breaks start() and makes it silently autostart. (described in the commit message)